manifests/vhost.pp

```diff
@@ -2335,10 +2336,49 @@
         include apache::mod::authz_groupfile
       }
 
+      if 'options' in $directory {
+        if !('-ExecCGI' in $directory['options']) {
+          if 'ExecCGI' in $directory['options'] {
+            case $apache::mpm_module {
+              'prefork': {
+                include apache::mod::cgi
+              }
+              'worker': {
+                include apache::mod::cgid
+              }
+              default: {
+                # do nothing
+              }
+            }
+          }
+        }
+      }
+
+      if 'dav' in $directory {
+        include apache::mod::dav
+        if $directory['dav'] == 'On' {
```

Comment:
I'm not sure we really enforce this to be `On`. Could be `on` or a boolean too, right? Would it be better to reverse the logic and first check for `svn` and then:
```suggestion
        if bool2httpd($directory['dav']) == 'On' {
```